.github/workflows/linting.yml

```
@@ -0,0 +1,33 @@
+name: PHP Linting (WordPress Coding Standards, PHPCS/PHPCBF)
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
```

```suggestion
on: [push, pull_request]
```
Bit of a nitpick but I think this is a nice simplification. By my undersanding of the docs, the only thing this brings in is running the CI on tags but I don't think that's an issue.